plugin-server/tests/postgres/vm.test.ts

```diff
@@ -1124,9 +1124,7 @@ describe('vm tests', () => {
             expect(fetch).toHaveBeenCalledWith('https://export.com/results.json?query=otherEvent2&events=2')
 
             // adds exportEventsWithRetry job and onEvent function
-            expect(Object.keys(vm.tasks.job)).toEqual(
```

Comment:
non-blocking

I don't see where this would have changed with the query update, to you know why it's changed?

Reply:
Yeah it's weird, this is failing which is related to a change in `master` (plugin tests are broken there).

Wondering if the recent GH issues have actions going 🍌 s

Reply:
Deleted the previous stupid comment. Tests were broken on master given I disabled historical exports quickly, hence this